cmd/miniooni/main.go

```diff
@@ -210,29 +204,36 @@ func main() {
 	}
 
 	if !globalOptions.noBouncer {
+		log.Info("Looking up OONI backends")
 		if err := sess.MaybeLookupBackends(ctx); err != nil {
 			log.WithError(err).Fatal("cannot lookup OONI backends")
 		}
 	}
 	if !globalOptions.noGeoIP {
+		log.Info("Looking up your location")
```

Comment:
After having used the program for a while, I think it's better to say something like `... you IP, country, and ISP name`. What do you think?